modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-core-platform-alerting-rules_{context}"]
+= Modifying core platform alerting rules
+
+As a cluster administrator, you can modify core platform alerting rules to suit your needs. For example, you can tweak an alert expression or change the severity label of an alert.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have enabled Technology Preview features, and all nodes in the cluster are ready.
+
+
+.Procedure
+
+. Create a new YAML configuration file named `example-modified-alerting-rule.yaml` in the `openshift-monitoring` namespace.
+
+. Add an `AlertRelabelConfig` resource to the YAML file. 
+The following example modifies the `severity` setting to `critical` for the default platform `watchdog` alerting rule: 
++
+[source,yaml]
+----
+apiVersion: monitoring.openshift.io/v1alpha1
+kind: AlertRelabelConfig
+metadata:
+  name: watchdog
+  namespace: openshift-monitoring
+spec:
+  configs:
+  - sourceLabels: [alertname,severity] <1>
+    regex: "Watchdog;none" <2>
+    targetLabel: severity <3>
+    replacement: critical <4>
+    action: Replace <5>
+----
+<1> The source labels for the values to be modified.
+<2> The name of the alerting rule you want to modify.
+<3> The target label of the value you want to modify.
+<4> The new value to replace the target label.
+<5> The relabel action that replaces the old value with the the new value based on regex matching.
+The default action is `Replace`.
+Other possible values are `Keep`, `Drop`, `Hashmod`, `Labelmap`, `Labeldrop`, and `Labelkeep`.
```

Comment:
with the fix from https://bugzilla.redhat.com/show_bug.cgi?id=2100312#c9
Other possible values are `Keep`, `Drop`, `HashMod`, `LabelMap`, `LabelDrop`, and `LabelKeep`.

Reply:
fixed